lib/rucio/rse/protocols/webdav.py

```diff
@@ -36,13 +36,32 @@
 from xml.parsers import expat
 
 import requests
-from progressbar import ProgressBar
+
 from requests.adapters import HTTPAdapter
 from urllib3.poolmanager import PoolManager
 
 from rucio.common import exception
 from rucio.rse.protocols import protocol
 
+try:
+    from progressbar import ProgressBar
+except ImportError:
+    # make progressbar an optional dependency
+    class ProgressBar:
```

Comment:
Why do we have progressbar here at all? Do we have it in any of the other protocols?

Reply:
It's only used in webdav. I'm not sure why that protocol uses it but none of the others do.

Reply:
@mlassnig any idea here? I think we should remove this.

Reply:
o.O no idea.

i'd rather have a progressbar that is universally usable for all protocols

Reply:
Exactly, but that should happen outside the protocol and not within.